### ISSUE TITLE
An error occurs in the "kintone-cli dev" command of the plugin

### ISSUE THREAD

**Issue by Yanchi4425** [kintone/kintone-cli#3]
When developing a plugin with "kintone-cli dev" command, the following error occurs.

```
C:\dev\kintone-customization-project>kintone-cli dev --app-name SamplePlugin --watch
Building distributed file...
Starting local webserver...

Please open this link in your browser to trust kintone Plugin files:

   https://172.18.93.81:8000

Then, press any key to continue:


Building plugin...
Invalid manifest.json:
- ".desktop.js[0]" should match format "https-url"
- ".desktop.js[0]" should match format "relative-path"
- ".desktop.js[0]" file size should be <= 20MB
- ".desktop.js[0]" should match some schema in anyOf
- ".mobile.js[0]" should match format "https-url"
- ".mobile.js[0]" should match format "relative-path"
- ".mobile.js[0]" file size should be <= 20MB
- ".mobile.js[0]" should match some schema in anyOf
Failed: Invalid manifest.json
Error: Invalid manifest.json
    at throwIfInvalidManifest (C:\dev\kintone-customization-project\node_modules\@kintone\plugin-packer\src\cli.js:131:11)
    at Promise.resolve.then (C:\dev\kintone-customization-project\node_modules\@kintone\plugin-packer\src\cli.js:43:7)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:832:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
Error: ENOENT: no such file or directory, rename 'SamplePlugin/dist/undefined' -> 'SamplePlugin/dist/private.ppk'
```

At that time, the manifest file is in the same directory as "node_modules" and looks like this:

```json
{
    "manifest_version": 1,
    "version": 1,
    "type": "APP",
    "icon": "SamplePlugin/icon.png",
    "name": {
        "en": "SamplePlugin"
    },
    "description": {
        "en": "Kintone Plugin"
    },
    "desktop": {
        "js": [
            "https://172.18.93.81:8000/SamplePlugin/dist/SamplePlugin.min.js"
        ],
        "css": []
    },
    "mobile": {
        "js": [
            "https://172.18.93.81:8000/SamplePlugin/dist/SamplePlugin.min.js"
        ]
    }
}
```

I tried this on Windows and Mac with the same results.
### Windows10
- Node: v10.16.0
- npm: v6.9.0

### MacOS
- Node: ?
- npm: v6.9.0

**Comment by miyass** [comment]
@Yanchi4425 
We've also checked this error. 
Maybe this error is caused by url validation.

We will fix this error. Thank you!

**Comment by oe-trader** [comment]
@miyass 
I still have the same problem.
Is it not still fixed in v0.1.1?

**Comment by miyass** [comment]
@Yanchi4425 @oe-trader 
This error was fixed in v0.1.2.
Thank you!

Note :
Please note that html file and img file for icon are not changed automatically. 
Only js/css files are changed.
